cumulusci.rb

```diff
@@ -3,9 +3,10 @@ class Cumulusci < Formula
 
   desc "Python framework for building automation for Salesforce projects"
   homepage "https://github.com/SFDO-Tooling/CumulusCI"
-  url "https://files.pythonhosted.org/packages/80/8e/054e92f3edce2af2723872ceffedfcadcafab1c81952e49c12676db99fbf/cumulusci-3.64.0.tar.gz"
-  sha256 "48ba2bb84abbc4da22747ba48d2215e055e1786970aa4463a4424d63a0940a6e"
-  head "https://github.com/SFDO-Tooling/CumulusCI.git"
+  url "https://files.pythonhosted.org/packages/70/74/b0b7168887d7adc5d93adda3776e71287db9c004e5c11e332fce60a9db1b/cumulusci-3.65.0.tar.gz"
+  sha256 "e2bd8bef745122147b770d0705699f1e1474678ab8a80051eb9ac3548f57c9ce"
+  license "BSD-3-Clause"
```

Comment:
Not seen this added before, but since this is the license for CumulusCI I believe its OK.